ple/games/tetris_engine.py

```diff
@@ -0,0 +1,518 @@
+# Modified from Tetromino by lusob luis@sobrecueva.com
+# http://lusob.com
+# Released under a "Simplified BSD" license
+
+import random, time, pygame, sys
+from pygame.locals import *
+
+FPS = 25
```

Comment:
can you move all these constants into the class `GameState`? such that you call it via self.CONSTANT_NAME